packages/jazz-inspector-element/src/main.ts

```diff
@@ -0,0 +1,3 @@
+if (typeof window !== "undefined" && process.env.NODE_ENV === "development") {
+  import("./element.js");
+}
```

Comment:
Doing this to register the element and load react only in the browser and in dev mode.

The check on window is required to work on svelte-kit without asking the user to do checks.